### PR TITLE
Missing annotations on websockets module.

### PR DESCRIPTION
**Pull request by marcosschroh**
Command to run: `mypy starlette/websockets.py --disallow-untyped-defs --ignore-missing-imports --follow-imports=silent`

### CODE REVIEW

File: starlette/websockets.py [hunk 1]
```diff
@@ -110,52 +113,54 @@ def query_params(self) -> QueryParams:
         else:
             raise RuntimeError('Cannot call "send" once a close message has been sent.')
 
-    async def accept(self, subprotocol=None):
+    async def accept(
+        self, subprotocol: typing.Optional[typing.List[str]] = None
```
Comment (lovelydinosaur) [inline starlette/websockets.py]: Let's use implied optionals `typing.List[str]= None`

**Comment by lovelydinosaur** [comment]
Cool - there's only one very minor comment on this. If we can use implied optionals in that case, then I think this is good to go.

File: starlette/websockets.py [hunk 2]
```diff
@@ -13,26 +16,26 @@ class WebSocketState(enum.Enum):
...
 
 class WebSocketDisconnect(Exception):
-    def __init__(self, code=1000):
+    def __init__(self, code: int = 1000) -> None:
         self.code = code
 
 
 class WebSocket(Mapping):
-    def __init__(self, scope, receive=None, send=None):
+    def __init__(self, scope: Scope, receive: Receive, send: Send) -> None:
```
Comment (lovelydinosaur) [inline starlette/websockets.py:24]: Is it possible to leave `send` and `receive` as optionals for now, or does that introduce errors?

Comment (marcosschroh) [reply starlette/websockets.py:24]: If we allow optional like before we have ` error: "None" not callable`. Anyway, does it make sense that `receive` and `send` can be optional? If for some reason any of them is `None` the server will crash.

**Pull request merged**